spec/rails_helper.rb

```diff
@@ -125,6 +125,7 @@
     Course::IDENTIFIERS.each do |identifier|
       FactoryBot.create(identifier)
     end
+    Rails.application.load_tasks
```

Comment:
Can this get scoped to just rake specs, so they're not loaded when the other specs are individually run?